empathylink.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
+    settings = get_settings
+    @buddies = settings.hunting_buddies
+    @use_hodierna = DRC.get_settings.hashodierna
+    in_room = DRRoom.pcs
+    while true
+      in_room.each do | target |
+        if shouldheal(target)
+          touch(target)
+        end
+      end
+      pause 0.5
+      in_room = DRRoom.pcs
+    end
+  end
+
+  def populate_noheal_list
+    #UserVars.empathylink['xxxx'] = 2496928411
```

Comment:
After reading through the script, I think the value of this hash is the next time that person can be checked. I read 'noheal' as 'people to never heal'. Do you think this could be confusing to users of this script?

Reply:
I commented out the names that were in there and left them as an example. So, if you have a name here to never heal and the name was on your buddies list, no healing would occur.

Reply:
Added comment for clarification in script.